.github/workflows/deploy-template.yml

```diff
@@ -31,28 +31,24 @@ jobs:
       - name: Parse directories
         id: parse_directories
         run: |
+
           tag=${{ steps.tag_name.outputs.tag }}
-          if [ "${{ inputs.platform }}" == "protectli" ]; then
-            base_dir="protectli"
-            model=$(echo "$tag" | cut -d'_' -f1-3)
-            release=$(echo "$tag" | cut -d'_' -f4)
-          elif [ "${{ inputs.platform }}" == "novacustom" ]; then
-            base_dir="novacustom"
-            model=$(echo "$tag" | cut -d'_' -f1-3)
-            release=$(echo "$tag" | cut -d'_' -f4)
-          elif [ "${{ inputs.platform }}" == "pcengines" ]; then
-            base_dir="pcengines"
-            model="pcengines_apu2"
-            release=$(echo "$tag" | cut -d'_' -f3)
-          elif [ "${{ inputs.platform }}" == "msi" ]; then
-            base_dir="3mdeb/msi_$(echo "$tag" | cut -d'_' -f1-2)"
-            model=""
-            release=$(echo "$tag" | cut -d'_' -f3)
-          elif [ "${{ inputs.platform }}" == "hardkernel" ]; then
-            base_dir="3mdeb/hardkernel_$(echo "$tag" | cut -d'_' -f2-3)"
-            model=""
-            release=$(echo "$tag" | cut -d'_' -f4)
-          fi
+          # Extract platform model string (everything before the last underscore in the tag)
+          model=$(echo "$tag" | awk -F'_' '{OFS="_"; $(NF)=""; print $0}' | sed 's/_$//')
+          # Extract version string (everything after the last underscore in the tag)
+          release=$(echo "$tag" | awk -F'_' '{print $NF}')
+
+          # Use 3mdeb directory for community / DPP releases
+          case "${{ inputs.platform }}" in
+            msi|hardkernel)
```

Comment:
theoretically also dell and intel minnowboard

Reply:
BUt instead of having to update this file for each new board, we could give another input param to set whether it is community/dpp or not. Then only build.yml will be updated

Reply:
Technicaly speaking, PC Engines is DPP/community release, but does not follow this logic...

Reply:
:man_shrugging: